communication/remote.h

```diff
@@ -312,18 +335,32 @@ void remote_get_thrust_command(const remote_t* remote, thrust_command_t * comman
  * 
  * \param	remote			Remote structure (input)
  * \param	command			Attitude command (output)
+ * \param   scale			Scale (maximum output / max remote input)
+ */
+void remote_get_attitude_command_absolute_yaw(const remote_t* remote, attitude_command_t * command, float scale);
+
+
+/**
+ * \brief	Compute attitude command from the remote (absolute roll and pitch, integrated yaw)
+ * 
+ * \param	remote			Remote structure (input)
+ * \param 	ki_yaw			Integration factor for yaw (0.02 is ok) (input) 
```

Comment:
Fixed in https://github.com/lis-epfl/MAVRIC_Library/commit/2f0fa87c288685d8e16d6c05d3d98a2d27aa860e